infra/timer/Timer.hpp

```diff
@@ -85,24 +85,34 @@ namespace infra
         void ComputeNextTriggerTime() override;
     };
 
+    namespace details
```

Comment:
For consistency with other detail namespaces:
```suggestion
    namespace detail
```